README.md

```diff
@@ -163,3 +163,65 @@ Interested in Integrating SocialConnect, get in touch by filling this [form](htt
 ## 📣 Feedback
 
 **SocialConnect is in beta**! Help us improve by sharing feedback on your experience in the Github Discussion section. You can also open an issue or a PR directly on this repo.
+
+## FAQ
+
+<details>
+  <summary>What is an Identifier?</summary>
+
+Identifier is any string of text that a user can use to identify other user.
+
+For example, Phone number, Twitter handle, GitHub username etc...
+
+</details>
+
+<details>
+  <summary>What is an Issuer?</summary>
+
+Issuer is an entity that is willing to take the overhead of verifying a user's ownership of an identifier.
+
+</details>
+
+<details>
+  <summary>Does Issuer need to pay for gas?</summary>
+
+Yes, Issuer pays for gas every time it wants to attest a new user.
+
+</details>
+
+<details>
+  <summary>Does Issuer need to have ODIS quota?</summary>
+
+Yes, Issuer needs to have ODIS Quota to register and lookup users.
```

Comment:
This is recommended but not strictly required; recommended since it promotes interoperable identifiers. This isn't necessary when issuers do not care about identifiers that use ODIS peppers; in these cases issuers can also use proprietary prefixes/salts, which means giving up interoperability but provides additional security

Reply:
Does it mean that users can do the ODIS part themselves and then simply provide the `obfuscatedIdentifier` to be registered under the respective issuer?

Same for lookup.

Technically that would mean the `authSigner` will be the user requesting registration and the dApp would basically look up its balance and request to buy quota in-app, right?

Reply:
In the case of proprietary prefixes, wouldn't it be mandatory for issuers to have ODIS quota?

I am assuming the issuer wouldn't want its users to know the prefix and thus will have to perform the process of getting the `obfuscatedIdentifier` itself.